### PR TITLE
Change adapter to @adapts decorator.

### PR DESCRIPTION
**Pull request by thet**

### CODE REVIEW

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

File: news/3878.bugfix [hunk 1]
```diff
@@ -0,0 +1,2 @@
+Change adapter to @adapts decorator.
```
Comment (gforcada) [inline news/3878.bugfix:1]:
```suggestion
Change adapts to @adapter decorator.
```

😄

**Comment by thet** [comment]
@jenkins-plone-org please run jobs

**Review by davisagli — APPROVED** [review]

**Comment by mauritsvanrees** [comment]
Thanks!
I am surprised this was still in there. I though the old way already did not work anymore.

**Comment by thet** [comment]
@mauritsvanrees me too :)

**Pull request merged**